tests/commands/test_config.py

```diff
@@ -0,0 +1,92 @@
+import os
+import unittest
+import tempfile
+
+from click.testing import CliRunner
+from tutor.commands.config import *
+from tests.test import CONTEXT
+from tutor import config as tutor_config
+
+
+class ConfigTests(unittest.TestCase):
+    def test_config_help(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(config_command, ["--help"])
+        self.assertEqual(0, result.exit_code)
+        self.assertFalse(result.exception)
```

Comment:
Is this False or None?

Reply:
Didn't notice there was this function self.assertIsNone(result.exception). Change to use that instead for every related unit tests scenario.